Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizer.cs

```diff
@@ -0,0 +1,133 @@
+        }
+
+        /// <summary>
+        /// Displays the boundary as a quad primitive.
+        /// </summary>
+        private void AddQuad()
+        {
+            // Get the rectangular bounds.
+            Vector2 center = EdgeUtilities.InvalidPoint;
+            float angle = 0f;
+            float width = 0f;
+            float height = 0f;
+            if (!(bool)(BoundaryManager?.TryGetRectangularBoundsParams(out center, out angle, out width, out height)))
+            {
+                // No rectangular bounds, therefore do not render the quad.
+                return;
+            }
+            
+            // Render the rectangular bounds.
+            if (EdgeUtilities.IsValidPoint(center))
+            {
+                GameObject quad = GameObject.CreatePrimitive(PrimitiveType.Quad);
+                quad.transform.SetParent(transform);
+                quad.transform.Translate(new Vector3(center.x, 0.005f, center.y)); // Add fudge factor to avoid z-fighting
+                quad.transform.Rotate(new Vector3(90, -angle, 0));
+                quad.transform.localScale = new Vector3(width, height, 1.0f);
+                quad.GetComponent<Renderer>().sharedMaterial = inscribedRectangleMaterial;
+            }
+        }
+
+        /// <summary>
+        /// Displays the boundary as an array of spheres where spheres in the
+        /// bounds are a different color.
+        /// </summary>
+        private void AddIndicators()
+        {
+            // Get the rectangular bounds.
+            Vector2 centerRect = EdgeUtilities.InvalidPoint;
+            float angleRect = 0f;
+            float widthRect = 0f;
+            float heightRect = 0f;
+            if (!(bool)(BoundaryManager?.TryGetRectangularBoundsParams(out centerRect, out angleRect, out widthRect, out heightRect)))
+            {
+                // If we have no boundary manager or rectangular bounds we will show no indicators
+                return;
+            }
+
+            const int indicatorCount = 20;
+            const float indicatorDistance = 0.2f;
+            const float dimension = indicatorCount * indicatorDistance;
+
+            Vector3 center = new Vector3(centerRect.x, 0f, centerRect.y);
+            Vector3 corner = center - (new Vector3(dimension, 0.0f, dimension) * 0.5f);
+
+            corner.y += 0.05f;
+            for (int xIndex = 0; xIndex < indicatorCount; ++xIndex)
+            {
+                for (int yIndex = 0; yIndex < indicatorCount; ++yIndex)
+                {
+                    Vector3 offset = new Vector3(xIndex * indicatorDistance, 0.0f, yIndex * indicatorDistance);
+                    Vector3 position = corner + offset;
+                    GameObject marker = GameObject.CreatePrimitive(PrimitiveType.Sphere);
+                    marker.transform.SetParent(transform);
+                    marker.transform.position = position;
+                    marker.transform.localScale = new Vector3(0.1f, 0.1f, 0.1f);
```

Comment:
`marker.transform.localScale = Vector3.one * 0.1f;`?

Is there any particular reason we're using 0.1f?

Reply:
goldilocks spheres (not to big not too small... just right :) )